ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -111,16 +120,43 @@ TorchCore::~TorchCore ()
   g_free (model_path);
 }
 
+/**
+ * @brief	Set the accelerator for the pytorch
+ */
+void
+TorchCore::setAccelerator (const char *accelerators)
+{
+  use_gpu = TRUE;
+  accelerator = parse_accl_hw (accelerators, torch_accl_support);
+  if (accelerator == ACCL_NONE)
+    goto use_gpu_ini;
+  if ((accelerator & (ACCL_CPU | ACCL_DEFAULT | ACCL_AUTO)) != 0)
+    use_gpu = FALSE;
+
+  g_message (" 0 gpu = %d", use_gpu);
```

Comment:
What's the meaning of a number followed by `gpu`?

Reply:
This was unnecessary logging, removed.